app/controllers/training_modules_users_controller.rb

```diff
@@ -31,6 +32,38 @@ def mark_exercise_complete
 
   private
 
+  def instructor_training_complete?
```

Comment:
This feature should not be in the same PR. I'd prefer to first merge the simplest possible version that will address the issue.

Reply:
Okay